default/stringtables/en.txt

```diff
@@ -4866,8 +4866,10 @@ At %planet%: depopulation is complete.
 EFFECT_CONC_CAMP_COMLETE_LABEL
 Ethnic Cleansing
 
-EFFECT_DRYDOCK_SHIP_REPAIR
-On %planet%: the %building% repaired the %ship%.
+EFFECT_DRYDOCK_SHIP_REPAIR_PARTIAL
+On %planet%: partial repairs were performed on the %ship% by the %building%.
```

Comment:
Needs to be worded the same way as complete, ie "the %building% partially repaired the %ship%." That's to ensure that fleet/shipnames are always clickable at the end of the line for consistency